src/test/java/persistence/sql/entity/LazyLoadingTest.java

```diff
@@ -0,0 +1,31 @@
+package persistence.sql.entity;
+
+import org.junit.jupiter.api.Test;
+import persistence.sql.ddl.Order;
+import persistence.sql.ddl.Review;
+
+import java.lang.reflect.Proxy;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LazyLoadingTest {
+    @Test
+    void testLazyLoadingProxy() {
+        final Order order = new Order(1L, "ORD123", new ArrayList<>());
```

Comment:
```suggestion
        final Order order = new Order(1L, "ORD123", new ArrayList<>(),  createProxyReviews());
```

`reviews.getFirst();`진짜로 접근할때 Proxy를 타는지 보기위해 위와 같이 할것같네요~